lib/engine/game/g_1825/game.rb

```diff
@@ -189,12 +189,38 @@ def game_phases
           },
         }.freeze
 
+        ALL_TRAINS_K3 = {
+          '2' => { distance: 2, price: 180, rusts_on: '5' },
+          '3' => { distance: 3, price: 300, rusts_on: '7' },
+          '4' => { distance: 4, price: 430 },
+          '5' => { distance: 5, price: 550 },
+          '3T' => { distance: 3, price: 370, available_on: '3' },
+          'U3' => {
+            distance: [{ 'nodes' => ['city'], 'pay' => 3, 'visit' => 3 },
+                       { 'nodes' => ['town'], 'pay' => 99, 'visit' => 99 }],
+            price: 410,
+            available_on: '3',
+          },
+          '6' => { distance: 7, price: 650 },
+          '4T' => { distance: 4, price: 480, available_on: '4' },
+          '2+2' => { distance: 2, price: 600, multiplier: 2, available_on: '4' },
+          '7' => { distance: 7, price: 720 },
+          '4+4E' => {
+            distance: [{ 'nodes' => ['city'], 'pay' => 4, 'visit' => 99 },
+                       { 'nodes' => ['town'], 'pay' => 0, 'visit' => 99 }],
+            price: 830,
+            multipler: 2,
+            available_on: '4a',
+          },
+        }.freeze
+
         def build_train_list(thash)
           thash.keys.map do |t|
             new_hash = {}
             new_hash[:name] = t
             new_hash[:num] = thash[t]
-            new_hash.merge!(ALL_TRAINS[t])
```

Comment:
is this constant mutation?

Reply:
I don't think so since it's just merging a constant hash into a new hash. I'm not changing any of the hash values being merged.